README.md

```diff
@@ -1,2 +1,310 @@
-# validator
-A tiny runtime library for type assertions
+# Validator
+
+A tiny library for run time type assertions.
+
+## Installation
+
+```bash
+npm install @mapbox/validator
+```
+
+## Usage
+
+### Basic
+The basic usage 
```

Comment:
I think a more extended description would be helpful here. What is a broad description of what is happening in the code below? That will give me context for understanding the more detailed documentation that follows.